src/calculateRentalCost.js

```diff
@@ -20,7 +20,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return rentalCost - 20;
+  } else if (days >= 7) {
+    return rentalCost - 50;
+  }
```

Comment:
what 50 and 20 mean? create vars

Reply:
ok
also I created a variable for price per day (40) 
I hope it'll ok